inductiva/fluids/wind_tunnel/README.md

```diff
@@ -31,11 +31,11 @@ import inductiva
 
 inductiva.api_key = "YOUR_API_KEY"
 
-# Url to a test vehicle in Inductiva Github repository
-vehicle_url = "https://raw.githubusercontent.com/inductiva/inductiva/main" \
+# Url to a test object in Inductiva Github repository
+object_url = "https://raw.githubusercontent.com/inductiva/inductiva/main" \
```

Comment:
Ok, then we can see if to make the path shorter we host it on our website or not... 
It could become "https://inductiva.ai/resources/vehicle.obj" for example...
Another possibility would be to use an url shortener service, but it becomes a bit more obfuscated, so let's avoid that for now.

Reply:
Yes, that's why I initially was against that solution that @sarmento suggested on hosting on the website, but we can reconsider giving the size of this url from github. But don't worry in this PR, leave it like this now. If we decide to change it will be a simple mechanical change to a url.